src/Select.js

```diff
@@ -0,0 +1,64 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import { theme } from 'styled-system'
+
+import { Box, Flex, Icon, Label } from '../src'
+
+const parseChevronIcon = isExpanded => {
+  const ChevronIcon = isExpanded
+    ? Icon.extend`
+        transform: rotate(180deg);
+      `
+    : Icon
+
+  return <ChevronIcon name="chevronDown" size={12} color="black" />
+}
+
+const SelectFlex = Flex.extend`
+  cursor: pointer;
+  height: 48px;
+  border-radius: ${theme('radius')};
+  border-width: 1px;
+  border-style: solid;
+  border-color: ${theme('colors.borderGray')};
+`
+
+const SelectBox = Box.extend`
+  cursor: pointer;
+`
+
+const SelectLabel = Label.extend`
+  cursor: pointer;
+`
+
+const Select = props => (
+  <SelectFlex justify="center" align="center">
+    <SelectBox pl={3}>
+      <Icon name="key" size={24} color="blue" />
+    </SelectBox>
+    <SelectBox pr={3}>
+      <SelectLabel fontSize={1} ml={2} align="left">
+        {props.children}
+      </SelectLabel>
+    </SelectBox>
+    <SelectBox pl={5} pr={3}>
+      {parseChevronIcon(props.isExpanded)}
+    </SelectBox>
+  </SelectFlex>
+)
+
+Select.displayName = 'Select'
+
+Select.propTypes = {
+  isExpanded: PropTypes.bool,
```

Comment:
Since dropdown components are *very* similar to the `<summary>` and `<details>` HTML elements, I tend to favor naming props like this after the HTML `open` attribute – see https://developer.mozilla.org/en-US/docs/Web/HTML/Element/details

Reply:
Thanks for the reference. Updated the prop name to `open` to match HTML native attribute.